### PR TITLE
Adding file upload progress bar ui

### PR DESCRIPTION
**Pull request by lmanohara**

### CODE REVIEW

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/application/template.jag [hunk 1]
```diff
@@ -86,21 +86,47 @@ jagg.template("application", function (inputs, outputs, jagg) {
 
             });
 
+            $('#fileupload').fileupload({
+
+                progressall: function (e, data) {
+                    var progress = parseInt(data.loaded / data.total * 100, 10);
```
Comment (danushkaf) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/application/template.jag]: Add a comment describing 100 and 10. Probably add comments

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/application/template.jag [hunk 2]
```diff
@@ -86,21 +86,47 @@ jagg.template("application", function (inputs, outputs, jagg) {
 
             });
 
+            $('#fileupload').fileupload({
+
```
Comment (punnadi) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/application/template.jag]: Blank lines

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/application/template.jag (hunk 1 above)
Comment (lmanohara) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/application/template.jag]: This is fixed in following commit.
https://github.com/wso2/app-cloud/pull/167/commits/6e2ea619f4ccbaa032aa59e465134ec81d28bf1a

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/application/template.jag (hunk 2 above)
Comment (lmanohara) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/application/template.jag]: This is fixed in master branch.

File: modules/jaggeryapps/appmgt/src/site/themes/default/js/vendor/jquery.ui.widget.js [hunk 3]
```diff
@@ -0,0 +1,572 @@
+/*! jQuery UI - v1.11.4+CommonJS - 2015-08-28
```
Comment (dakshika) [inline modules/jaggeryapps/appmgt/src/site/themes/default/js/vendor/jquery.ui.widget.js]: is this required ?

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/application/template.jag [hunk 4]
```diff
@@ -158,14 +184,16 @@ jagg.template("application", function (inputs, outputs, jagg) {
             <div class="form-group">
             <label>Upload Artifact: </label>
             <div class="input-group">
-                <input type="text" class="form-control" readonly>
+                <input id="uploadedFileName" type="text" class="form-control" readonly>
                 <span class="input-group-btn">
                    <span class="btn btn-primary btn-file">
-                       Upload&hellip; <input type="file" name="fileUpload" multiple>
+                       Upload&hellip; <input id="fileupload" type="file" name="fileUpload" multiple>
```
Comment (dakshika) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/application/template.jag]: what is hellip?

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/application/template.jag [hunk 5]
```diff
@@ -200,6 +228,8 @@ jagg.template("application", function (inputs, outputs, jagg) {
         </div>
       </div><!-- /.container -->
 
-      <script src="<%=jagg.getAbsoluteUrl(jagg.getThemeFile('js/custom/custom.js'))%>"></script>
+      <script src="<%=jagg.getAbsoluteUrl(jagg.getThemeFile('js/vendor/jquery.ui.widget.js'))%>"></script>
```
Comment (dakshika) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/application/template.jag]: AFAIK this plugin isn't required for default behavior? please double check

File: modules/jaggeryapps/appmgt/src/site/themes/default/js/vendor/jquery.ui.widget.js (hunk 3 above)
Comment (lmanohara) [inline modules/jaggeryapps/appmgt/src/site/themes/default/js/vendor/jquery.ui.widget.js]: Yes. This is required for upload jquery library. 
https://blueimp.github.io/jQuery-File-Upload/

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/application/template.jag (hunk 4 above)
Comment (lmanohara) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/application/template.jag]: hellip has been removed from the latest code.

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/application/template.jag (hunk 5 above)
Comment (lmanohara) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/application/template.jag]: This is required for upload jquery library. 
https://blueimp.github.io/jQuery-File-Upload/

**Pull request merged**